.github/workflows/run_tests.yaml

```diff
@@ -0,0 +1,26 @@
+on: push
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - run: |
+          pip install -r requirements.txt
+          wget https://github.com/broadinstitute/cromwell/releases/download/84/cromwell-84.jar
+          python watt.py -e cromwell-84.jar -t simple compress_file expect_fail -p $(nproc)
+      - run: |
```

Comment:
Why separate these into a separate run command? Does it run faster? The purpose of the tool was to make it easy to run everything with just one command

Reply:
I think for some reason I thought that separating them like this would make error (ie, unexpected successes) easier to diagnose.  But looking back at this I think you're right and there's no actual advantage.  will change to a one-liner.

Reply:
oh wait, I got confused. each "fail" needs to be tested separately, but you are right there doesn't need to be a separate run command.

Reply:
What do you mean that each fail needs to run separately? Even in the case where some tests fail due to Cromwell falling over, that should be captured just fine in the summary output

Reply:
we want the full automated test to fail if any of these tests pass, since we want them to fail (as in watt says they fail, not that watt expects cromwell to fail).  If they are all run in a single command, then that run of watt can fail even if only one of the listed tests is failing and all the others pass.  So the entire test can "pass" even though we actually had some tests that "succeeded" when they shouldn't have.  So to make sure that every one of these tests fails, we need to run them each separately.

Reply:
Oh I think I see what you're saying. If you have `test1` and `test2` both fail "unexpectedly" because something broke the tests, then Watt should exit with nonzero code to signify something is wrong. But to test that Watt itself is working properly in this way, it should be tested that each of them produce nonzero exit code separately, right? I.e. to prevent the case where `test2` silently passes whereas `test1` is failing to produce the "right" exit code.

Reply:
yup, exactly